code/item.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'tax_calculator'
+class Item
+  include TaxCalculator
+  attr_accessor :name, :cost, :quantity, :detail
+  def initialize(item)
+    @detail = item
+    assign
+  end
+
+  def assign
+    @cost = detail[-1].to_f
+    @quantity = detail[0].to_f
+    @name = detail[1..detail.length - 3].join(' ')
+  end
+
+  def display_name
+    name.gsub('imported ', '')
+  end
```

Comment:
@akavinash1994  in billing you find item name by `(item.imported? ? ' imported ' : ' ') + item.display_name  `
Could you update `def display_name` 
so it append `impoted` in display name if item is imported